File: Python/1048.py
sala = float(input())

if sala <= 400:
    novo_salario = sala + sala * 0.15
    reajuste_ganho = sala * 0.15
    percentual = 15
elif sala > 400 and sala <= 800:
    novo_salario = sala + sala * 0.12
    reajuste_ganho = sala * 0.12
    percentual = 12
elif sala > 800 and sala <= 1200:
    novo_salario = sala + sala * 0.10
    reajuste_ganho = sala * 0.10
    percentual = 10
elif sala > 1200 and sala <= 2000:
    novo_salario = sala + sala * 0.07
    reajuste_ganho = sala * 0.07
    percentual = 7
else:
    novo_salario = sala + sala * 0.04
    reajuste_ganho = sala * 0.04
    percentual = 4

print(f"Novo salario: {novo_salario:.2f}")
print(f"Reajuste ganho: {reajuste_ganho:.2f}")
print(f"Em percentual: {percentual} %")

File: C++/1048.cpp
#include <iostream>
#include <iomanip>

using namespace std;
 
int main() {
 
   double sala;
   cin >> sala ;
   
   if(sala<=400){
       cout << fixed << setprecision(2) << "Novo salario: " << sala+sala*0.15<< endl << "Reajuste ganho: " << sala*0.15 <<endl << "Em percentual: 15 %"<< endl;
   }
    else if(sala>400 && sala <= 800){
       cout << fixed << setprecision(2)<< "Novo salario: " << sala+sala*0.12<< endl << "Reajuste ganho: " << sala*0.12 <<endl << "Em percentual: 12 %"<< endl;
   }
    else if(sala>800 && sala <= 1200){
       cout << fixed << setprecision(2)<< "Novo salario: " << sala+sala*0.10<< endl << "Reajuste ganho: " << sala*0.10 <<endl << "Em percentual: 10 %"<< endl;
   }
    else if(sala>1200 && sala <= 2000){
       cout << fixed << setprecision(2)<< "Novo salario: " << sala+sala*0.07<< endl << "Reajuste ganho: " << sala*0.07 <<endl << "Em percentual: 7 %"<< endl;
   }
   else 
   {
     cout << fixed << setprecision(2)<< "Novo salario: " << sala+sala*0.04<< endl << "Reajuste ganho: " << sala*0.04 <<endl << "Em percentual: 4 %"<< endl;  
   }
   
 
    return 0;
}